src/measurement_kit/cmdline/meek_fronted_requests.cpp

```diff
@@ -0,0 +1,83 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include "../cmdline/cmdline.hpp"
+#include <measurement_kit/nettests.hpp>
+#include <measurement_kit/http.hpp>
+
+#include <iostream>
+
+namespace mk {
+namespace cmdline {
+namespace meek_fronted_requests {
+
+static const char *kv_usage =
+    "usage: measurement_kit meek_fronted_requests [-v] input_file \n";
```

Comment:
Note: now that #1156 has been merged, it would be optimal to refactor this file to be consistent with the new command line coding practices.